src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,24 @@
+package duke.tasks;
+
+import java.util.LinkedList;
+
+public class TaskList {
+    public final LinkedList<Task> allTasks;
+
+    /**
+     * Initialises a TaskList with allTasks.
+     *
+     * @param allTasks The LinkedList containing all existing tasks.
+     */
+    public TaskList(LinkedList<Task> allTasks) {
+        this.allTasks = allTasks;
+    }
+
+    /**
+     * Initialises an empty TaskList.
+     */
+    public TaskList() {
+        this.allTasks = new LinkedList<>();
+    }
+
+}
```

Comment:
As mentioned above, there could be more OOP by encapsulating various methods here, including `addTask`, `deleteTask`, `findTask`, etc.